middlewares/accesslog/logger_test.go

```diff
@@ -82,7 +128,7 @@ func TestLogger(t *testing.T) {
 		assert.Equal(t, fmt.Sprintf("%d", len(helloWorld)), tokens[7], printLogdata(logdata))
```

Comment:
I find the part up to and include this line somewhat hard to read. Maybe a combination of a `require`s and a switch statement could improve readability? WDYT?